gateway/main.go

```diff
@@ -45,7 +46,7 @@ func Run() {
 		log.Fatalf("failed loading gateway configuration, reason=%v", err)
 	}
 	if err := webappjs.ConfigureServerURL(); err != nil {
-		log.Fatal(err)
+		log.Warnf("failed configuring webappjs server URL, running gateway without it, reason=%v", err)
```

Comment:
We should not let errors pass silently. Troubleshooting this kind of errors in production or in proof of concepts is very hard. A container crashing is better than trying to figure out why a blank screen is appearing when accessing the Webapp.

Since the goal is allowing this error pass for development environments, you could use the value unknown from the common version package, which means the build is not generated from a build pipeline. Example:

```go
import "github.com/hoophq/hoop/common/version"
(...)

if isProductionBuild := version.Get().Version != "unknown"; isProductionBuild {
  if err := webappjs.ConfigureServerURL(); err != nil {
    log.Fatal(err)
  }
}
```

Reply:
I want to make sure we're aligned on the trade-offs here. This change could potentially increase our MTTR for troubleshooting issues in both POCs and production environments. If you're comfortable with that impact, please proceed